src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,54 @@
+package duke;
+
+import command.Command;
+
+/**
+ * Represents a personal assistant chatbot that responds to command line inputs.
+ *
+ * @author felix-ong
+ */
+public class Duke {
+    private Storage storage;
+    private TaskList tasks;
+
+    /**
+     * Constructor for Duke with default filepath.
+     */
+    public Duke() {
+        this.storage = new Storage("data/tasks.txt");
+        try {
+            this.tasks = new TaskList(this.storage.loadData());
+        } catch (DukeException e) {
+            this.tasks = new TaskList();
+        }
+    }
+    /**
+     * Constructor for Duke.
+     *
+     * @param filePath Relative path to the storage location.
+     */
+    public Duke(String filePath) {
+        this.storage = new Storage(filePath);
+        try {
+            this.tasks = new TaskList(this.storage.loadData());
+        } catch (DukeException e) {
+            this.tasks = new TaskList();
+        }
+    }
```

Comment:
There seems to be a duplicate of codes here. Just a recommendation, perhaps you can create a method for this? For example,
```suggestion
    public Duke() {
        this.setUp("data/tasks.txt);
    }
    
    public Duke(String filePath) {
        this.setUp(filePath);
    }
    
    private void setUp(String filePath) {
        this.storage = new Storage(filePath);
        try {
            this.tasks = new TaskList(this.storage.loadData());
        } catch (DukeException e) {
            this.tasks = new TaskList();
        }
    }
```